### ISSUE TITLE
Cannot `go get`

### ISSUE THREAD

**Issue by MysticRyuujin** [wealdtech/ethdo#14]
When I try to run `go get`

```
# github.com/wealdtech/go-eth2-wallet-nd
..\..\..\go\src\github.com\wealdtech\go-eth2-wallet-nd\account.go:228:20: impossible type assertion:
        *wallet does not implement "github.com/wealdtech/go-eth2-wallet-types".Wallet (wrong type for Accounts method)
                have Accounts() <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
                want Accounts(context.Context) <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
..\..\..\go\src\github.com\wealdtech\go-eth2-wallet-nd\account.go:231:20: impossible type assertion:
        *wallet does not implement "github.com/wealdtech/go-eth2-wallet-types".Wallet (wrong type for Accounts method)
                have Accounts() <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
                want Accounts(context.Context) <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
..\..\..\go\src\github.com\wealdtech\go-eth2-wallet-nd\account.go:240:11: cannot use w (type *wallet) as type "github.com/wealdtech/go-eth2-wallet-types".Wallet in assignment:
        *wallet does not implement "github.com/wealdtech/go-eth2-wallet-types".Wallet (wrong type for Accounts method)
                have Accounts() <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
                want Accounts(context.Context) <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
..\..\..\go\src\github.com\wealdtech\go-eth2-wallet-nd\wallet.go:151:2: cannot use w (type *wallet) as type "github.com/wealdtech/go-eth2-wallet-types".Wallet in return argument:
        *wallet does not implement "github.com/wealdtech/go-eth2-wallet-types".Wallet (wrong type for Accounts method)
                have Accounts() <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
                want Accounts(context.Context) <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
..\..\..\go\src\github.com\wealdtech\go-eth2-wallet-nd\wallet.go:175:2: cannot use wallet (type *wallet) as type "github.com/wealdtech/go-eth2-wallet-types".Wallet in return argument:
        *wallet does not implement "github.com/wealdtech/go-eth2-wallet-types".Wallet (wrong type for Accounts method)
                have Accounts() <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
                want Accounts(context.Context) <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
..\..\..\go\src\github.com\wealdtech\go-eth2-wallet-nd\wallet.go:268:11: cannot use w (type *wallet) as type "github.com/wealdtech/go-eth2-wallet-types".Wallet in assignment:
        *wallet does not implement "github.com/wealdtech/go-eth2-wallet-types".Wallet (wrong type for Accounts method)
                have Accounts() <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
                want Accounts(context.Context) <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
..\..\..\go\src\github.com\wealdtech\go-eth2-wallet-nd\wallet.go:317:11: cannot use w (type *wallet) as type "github.com/wealdtech/go-eth2-wallet-types".Wallet in assignment:
        *wallet does not implement "github.com/wealdtech/go-eth2-wallet-types".Wallet (wrong type for Accounts method)
                have Accounts() <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
                want Accounts(context.Context) <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
..\..\..\go\src\github.com\wealdtech\go-eth2-wallet-nd\wallet.go:397:14: cannot use ext.Wallet (type *wallet) as type "github.com/wealdtech/go-eth2-wallet-types".Wallet in assignment:
        *wallet does not implement "github.com/wealdtech/go-eth2-wallet-types".Wallet (wrong type for Accounts method)
                have Accounts() <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
                want Accounts(context.Context) <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
..\..\..\go\src\github.com\wealdtech\go-eth2-wallet-nd\wallet.go:406:12: cannot use ext.Wallet (type *wallet) as type "github.com/wealdtech/go-eth2-wallet-types".Wallet in return argument:
        *wallet does not implement "github.com/wealdtech/go-eth2-wallet-types".Wallet (wrong type for Accounts method)
                have Accounts() <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
                want Accounts(context.Context) <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
# github.com/wealdtech/go-eth2-wallet-hd
..\..\..\go\src\github.com\wealdtech\go-eth2-wallet-hd\account.go:238:20: impossible type assertion:
        *wallet does not implement "github.com/wealdtech/go-eth2-wallet-types".Wallet (wrong type for Accounts method)
                have Accounts() <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
                want Accounts(context.Context) <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
..\..\..\go\src\github.com\wealdtech\go-eth2-wallet-hd\account.go:241:20: impossible type assertion:
        *wallet does not implement "github.com/wealdtech/go-eth2-wallet-types".Wallet (wrong type for Accounts method)
                have Accounts() <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
                want Accounts(context.Context) <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
..\..\..\go\src\github.com\wealdtech\go-eth2-wallet-hd\account.go:250:11: cannot use w (type *wallet) as type "github.com/wealdtech/go-eth2-wallet-types".Wallet in assignment:
        *wallet does not implement "github.com/wealdtech/go-eth2-wallet-types".Wallet (wrong type for Accounts method)
                have Accounts() <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
                want Accounts(context.Context) <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
..\..\..\go\src\github.com\wealdtech\go-eth2-wallet-hd\wallet.go:183:2: cannot use w (type *wallet) as type "github.com/wealdtech/go-eth2-wallet-types".Wallet in return argument:
        *wallet does not implement "github.com/wealdtech/go-eth2-wallet-types".Wallet (wrong type for Accounts method)
                have Accounts() <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
                want Accounts(context.Context) <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
..\..\..\go\src\github.com\wealdtech\go-eth2-wallet-hd\wallet.go:220:2: cannot use w (type *wallet) as type "github.com/wealdtech/go-eth2-wallet-types".Wallet in return argument:
        *wallet does not implement "github.com/wealdtech/go-eth2-wallet-types".Wallet (wrong type for Accounts method)
                have Accounts() <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
                want Accounts(context.Context) <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
..\..\..\go\src\github.com\wealdtech\go-eth2-wallet-hd\wallet.go:244:2: cannot use wallet (type *wallet) as type "github.com/wealdtech/go-eth2-wallet-types".Wallet in return argument:
        *wallet does not implement "github.com/wealdtech/go-eth2-wallet-types".Wallet (wrong type for Accounts method)
                have Accounts() <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
                want Accounts(context.Context) <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
..\..\..\go\src\github.com\wealdtech\go-eth2-wallet-hd\wallet.go:358:11: cannot use w (type *wallet) as type "github.com/wealdtech/go-eth2-wallet-types".Wallet in assignment:
        *wallet does not implement "github.com/wealdtech/go-eth2-wallet-types".Wallet (wrong type for Accounts method)
                have Accounts() <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
                want Accounts(context.Context) <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
..\..\..\go\src\github.com\wealdtech\go-eth2-wallet-hd\wallet.go:450:14: cannot use ext.Wallet (type *wallet) as type "github.com/wealdtech/go-eth2-wallet-types".Wallet in assignment:
        *wallet does not implement "github.com/wealdtech/go-eth2-wallet-types".Wallet (wrong type for Accounts method)
                have Accounts() <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
                want Accounts(context.Context) <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
..\..\..\go\src\github.com\wealdtech\go-eth2-wallet-hd\wallet.go:459:12: cannot use ext.Wallet (type *wallet) as type "github.com/wealdtech/go-eth2-wallet-types".Wallet in return argument:
        *wallet does not implement "github.com/wealdtech/go-eth2-wallet-types".Wallet (wrong type for Accounts method)
                have Accounts() <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
                want Accounts(context.Context) <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
..\..\..\go\src\github.com\wealdtech\go-eth2-wallet-hd\wallet.go:513:11: cannot use w (type *wallet) as type "github.com/wealdtech/go-eth2-wallet-types".Wallet in assignment:
        *wallet does not implement "github.com/wealdtech/go-eth2-wallet-types".Wallet (wrong type for Accounts method)
                have Accounts() <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
                want Accounts(context.Context) <-chan "github.com/wealdtech/go-eth2-wallet-types".Account
..\..\..\go\src\github.com\wealdtech\go-eth2-wallet-hd\wallet.go:513:11: too many errors
```

**Comment by MysticRyuujin** [comment]
I just tried this again, got new error:
```
# github.com/wealdtech/ethdo/cmd
..\..\..\go\src\github.com\wealdtech\ethdo\cmd\root.go:280:15: wallet.Unlock undefined (type "github.com/wealdtech/go-eth2-wallet-types".Wallet has no field or method Unlock)
..\..\..\go\src\github.com\wealdtech\ethdo\cmd\root.go:284:15: wallet.Lock undefined (type "github.com/wealdtech/go-eth2-wallet-types".Wallet has no field or method Lock)
..\..\..\go\src\github.com\wealdtech\ethdo\cmd\root.go:286:2: not enough arguments to return
..\..\..\go\src\github.com\wealdtech\ethdo\cmd\root.go:286:15: wallet.AccountByName undefined (type "github.com/wealdtech/go-eth2-wallet-types".Wallet has no field or method AccountByName)
..\..\..\go\src\github.com\wealdtech\ethdo\cmd\root.go:316:38: not enough arguments in call to wallet.Accounts
        have ()
        want (context.Context)
..\..\..\go\src\github.com\wealdtech\ethdo\cmd\root.go:365:13: account.IsUnlocked undefined (type "github.com/wealdtech/go-eth2-wallet-types".Account has no field or method IsUnlocked)
..\..\..\go\src\github.com\wealdtech\ethdo\cmd\root.go:370:2: not enough arguments to return
..\..\..\go\src\github.com\wealdtech\ethdo\cmd\root.go:370:16: account.Sign undefined (type "github.com/wealdtech/go-eth2-wallet-types".Account has no field or method Sign)
..\..\..\go\src\github.com\wealdtech\ethdo\cmd\validatorexit.go:146:16: account.Unlock undefined (type "github.com/wealdtech/go-eth2-wallet-types".Account has no field or method Unlock)
..\..\..\go\src\github.com\wealdtech\ethdo\cmd\walletcreate.go:59:27: not enough arguments in call to nd.CreateWallet
        have (string, "github.com/wealdtech/go-eth2-wallet-types".Store, *keystorev4.Encryptor)
        want (context.Context, string, "github.com/wealdtech/go-eth2-wallet-types".Store, "github.com/wealdtech/go-eth2-wallet-types".Encryptor)
..\..\..\go\src\github.com\wealdtech\ethdo\cmd\walletcreate.go:59:27: too many errors
```

**Comment by mcdee** [comment]
Some of the dependencies underlying ethdo are undergoing change, which results in master being unstable.  Pulling the latest tagged version should work.

**Comment by MysticRyuujin** [comment]
I wish I knew how to do that on windows haha

I tried searching Google for instructions but it makes absolutely no sense to me, and nobody ELI5

The whole GOPATH and environment variables that GO uses are a pain in the butt.

**Comment by mcdee** [comment]
I don't have a windows box to hand but you can try:

```
set GO111MODULE=on
go get github.com/wealdtech/ethdo
```

from a command prompt and that may work.

**Comment by MysticRyuujin** [comment]
Nope, it does not work, and I tried that, and 100 other things.

Windows always complains any time I try to specify a tag:
```
set GO111MODULE=on
go get github.com/wealdtech/ethdo@latest
go: cannot use path@version syntax in GOPATH mode
```

That leads me down a rabbit hole of creating GO environments, and other more complicated and frustrating things

**Comment by mcdee** [comment]
One of the changes coming is pre-compiled windows binaries, which should help.

**Comment by mcdee** [comment]
Version 1.5.0 has been released, which includes pre-built binaries in the releases page.

(It should also compile form source now, with all of the dependencies aligned).
